examples/certification/certified-counter/src/backend/Cargo.toml

```diff
@@ -10,8 +10,7 @@ crate-type = ["cdylib"]
 anyhow.workspace = true
 candid.workspace = true
 ic-cdk.workspace = true
-ic-cdk-macros.workspace = true
-ic-certification.workspace = true
+ic-certification = { workspace = true, features = ["serde"] }
```

Comment:
This example was previously broken because of this, and the changes in the next file. The changes in this PR mean we will catch this in the CI pipeline next time.